files/scripts/updateAndRebootCron

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Run this as a cron job:
+#  sudo cp updateAndRebootCron /etc/cron.d/updateAndRebootCron
+#  sudo chmod 0644 /etc/cron.d/updateAndRebootCron
+#
+# Some weirdness about Debian cron.weekly:
+# * file must be owned by root, at most 0755
+#
+
+1 1 * * 0 root (date && apt update && apt upgrade -y && systemctl reboot) > /var/log/updateAndRebootCron.log 2>&1
```

Comment:
I think you may want to force a non interactive apt run, sometimes it might want to update grub and if running on console through cron would fail, something like

`~/planX/cloud-automation/ grep noninteractive tf_files/aws/modules/utility-vm/cloud.tf
sudo DEBIAN_FRONTEND='noninteractive' apt-get -y -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' upgrade| sudo tee --append /var/log/bootstrapping_script.log`